e2e-test/src/test/features/NormalizeErrorScenarios.feature

```diff
@@ -0,0 +1,56 @@
+    Then Navigate to the properties page of plugin: "Transpose"
+    Then Click on the Validate button
+    Then Verify mandatory property error for below listed properties:
+      | fieldMapping      |
+      | fieldNormalizing  |
+
+  Scenario:Verify Normalize plugin validation errors for mandatory output schema
+    Given Open Datafusion Project to configure pipeline
+    When Expand Plugin group in the LHS plugins list: "Transform"
+    When Select plugin: "Transpose" from the plugins list as: "Transform"
+    Then Navigate to the properties page of plugin: "Transpose"
+    Then Click on the Validate button
+    Then Verify that the Plugin is displaying an error message: "errorMessageOutputSchema" on the header
+
+  Scenario:Verify Normalize plugin validation errors for invalid output schema
+    Given Open Datafusion Project to configure pipeline
+    When Expand Plugin group in the LHS plugins list: "Transform"
+    When Select plugin: "Transpose" from the plugins list as: "Transform"
+    Then Navigate to the properties page of plugin: "Transpose"
+    Then Enter Normalize plugin FieldsMapping "normalizeGcsInvalidFieldsMapping"
+    Then Enter Normalize plugin FieldsNormalize "invalidFieldsToBeNormalizedGCS"
+    Then Enter outputSchema "normalizeGcsInvalidOutputSchema"
+    Then Click on the Validate button
+    Then Delete the output schema added
+    Then Click on the Validate button
+    Then Verify that the Plugin is displaying an error message: "errorMessageInvalidOutputSchema" on the header
+
+  Scenario:Verify Normalize plugin validation errors for incorrect data in field mapping
+    Given Open Datafusion Project to configure pipeline
+    When Expand Plugin group in the LHS plugins list: "Transform"
+    When Select plugin: "Transpose" from the plugins list as: "Transform"
+    Then Navigate to the properties page of plugin: "Transpose"
+    Then Enter Normalize plugin FieldsMapping "normalizeGcsInvalidFieldsMapping"
+    Then Enter Normalize plugin FieldsNormalize "invalidFieldsToBeNormalizedGCS"
+    Then Enter outputSchema "normalizeGcsInvalidOutputSchema"
+    Then Click on the Validate button
+    Then Verify Normalize plugin in-line error message for incorrect field mapping: "mappingField"
+
+  @PLUGIN-1228
+  Scenario:Verify Normalize plugin validation errors for incorrect data in Normalize fields
+    Given Open Datafusion Project to configure pipeline
+    When Expand Plugin group in the LHS plugins list: "Transform"
+    When Select plugin: "Transpose" from the plugins list as: "Transform"
+    Then Navigate to the properties page of plugin: "Transpose"
+    Then Enter Normalize plugin FieldsMapping "normalizeGcsInvalidFieldsMapping"
+    Then Enter Normalize plugin FieldsNormalize "invalidFieldsToBeNormalizedGCS"
+    Then Enter outputSchema "normalizeGcsInvalidOutputSchema"
+    Then Click on the Validate button
+    Then Verify Normalize plugin in-line error message for incorrect field to normalize: "attributeType"
```

Comment:
This step can be added at the end of above scenario... as all other steps are same. And remove this scenario

Reply:
As discussed, there is a bug hence cannot be removed.